edit/monitoring.coreos.com.receiver/types/opsgenie.vue

```diff
@@ -151,7 +156,7 @@ export default {
             <div class="row responder">
               <div class="col span-6">
                 <span v-if="isView">{{ typeLabel(scope.row.value.type) }}</span>
-                <Select v-else v-model="scope.row.value.type" :mode="mode" label="Type" :options="TYPES" />
+                <LabeledSelect v-else v-model="scope.row.value.type" :mode="mode" label="Type" :options="TYPES" />
```

Comment:
Can we switch this back to a Select? The label 'Type' is already shown above - by adding it, the height of the two fields is different and looks odd - i.e. we have  LabelSelect next to a Select (unlabelled)